com.wamas.ide.launching/src/com/wamas/ide/launching/validation/LcDslValidator.xtend

```diff
@@ -362,7 +365,7 @@ class LcDslValidator extends AbstractLcDslValidator {
 			error("Test class " + className + " must point to an existing class in project " + javaProject, cfg.test, LC.testConfig_Class)
 		} else if (!Flags.isPublic(type.flags)) {
 			error("Test class " + className + " must be public.", cfg.test, LC.testConfig_Class)
-		} else if (type.methods.stream().filter([method | Flags.isPublic(method.flags)]).flatMap([method | method.annotations.stream]).noneMatch([annotation | annotation.elementName.equals("Test")])) {
+		} else if (type.methods.stream().filter([method | Flags.isPublic(method.flags)]).flatMap([method | method.annotations.stream]).noneMatch([annotation | TEST_ANNOTATION_NAMES.contains(annotation.elementName)])) {
```

Comment:
Would be good to adjust the error message as well.

Reply:
I have included the list in the error message.